requirements.txt

```diff
@@ -2,4 +2,5 @@ scipy>=0.11
 numpy>=1.3
 pandas
 matplotlib
-libpysal>=4.0.0
\ No newline at end of file
+libpysal>=4.0.0
+giddy>=2.2.2
```

Comment:
Why is giddy a dependency for this?

Reply:
The space time interaction module uses [giddy's utility functions](https://github.com/weikang9009/pointpats/blob/spacetime/pointpats/spacetime.py#L17) like [`giddy.util. shuffle_matrix`](https://github.com/weikang9009/pointpats/blob/spacetime/pointpats/spacetime.py#L356)

Reply:
Can we in-line the shuffle_matrix in this module rather than introduce a cross-package dependency?

Reply:
Sure. Sounds like a good idea.